### PR TITLE
Obsolete devel-doc package

### PR DESCRIPTION
**Pull request by imobach**

### CODE REVIEW

File: package/yast2-packager.spec [hunk 1]
```diff
@@ -83,6 +83,8 @@ Recommends:     yast2-registration
 # force *-webpin subpackage removal at upgrade
 Obsoletes:      yast2-packager-webpin < %version
```
Comment (mchf) [inline package/yast2-packager.spec]: NP: why empty line?

Comment (imobach) [inline package/yast2-packager.spec]: To make clear that it has nothing to do with the `-webpin` thingie (although is obvious).

**Comment by mchf** [comment]
LGTM

**Comment by imobach** [comment]
Thanks!

**Pull request merged**